### PR TITLE
Update init.sh

### PR DESCRIPTION
**Pull request by Dimokus88**
Added Decloud Node Lab snapshot provider

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Please make sure to update chart version each time you update anything in the chart.
I'm Chart.yaml file.

**Comment by Dimokus88** [comment]
> Please make sure to update chart version each time you update anything in the chart.
> I'm Chart.yaml file.

Maybe you mean https://github.com/akash-network/helm-charts/blob/main/charts%2Fakash-node%2Fvalues.yaml ?

**Comment by Dimokus88** [comment]
> Please make sure to update chart version each time you update anything in the chart. I'm Chart.yaml file.

done

File: charts/akash-node/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ version: 11.1.0
 # Refs
 # https://github.com/akash-network/node/releases
 # https://github.com/akash-network/net/blob/main/mainnet/version.txt
-appVersion: 0.36.0
+appVersion: 0.36.1
```
Comment (ghost) [inline charts/akash-node/Chart.yaml]: Please keep 0.36.0 here as is.
There were no new node releases yet. Akash network is still running at 0.36.0

https://github.com/akash-network/node/releases
https://github.com/akash-network/net/blob/main/mainnet/version.txt

**Review by ghost — CHANGES_REQUESTED** [review]

File: charts/akash-node/Chart.yaml (hunk 1 above)
Comment (Dimokus88) [reply charts/akash-node/Chart.yaml]: @andy108369 fix it!

**Comment by ghost** [comment]
Thank you @Dimokus88 , I've fixed up multiple commits into one, with small fixes (proper commit naming).
Will be followed up here => https://github.com/akash-network/helm-charts/pull/292
Your commit is authored by you (you can see in the `Commits` tab there)

**Pull request closed without merging**